### DISCUSSION TITLE
[Request] Pelican LXC

### DISCUSSION THREAD

**Discussion by pafade89** [tteck/Proxmox#3681] [Ideas]
I've already been using many of your very helpful scripts and have now found something that could also make life easier for many as a script:

[https://pelican.dev/](https://pelican.dev/)

Pelican is the ultimate, free game server control panel offering high flying security. It's a breeze to manage servers with sleek and user-friendly interface.

**Comment by tteck** [comment]
I'm not motivated to work on game related scripts.

**Reply by pafade89** [reply]
Okay but thanks for your quick reply.
